srcpkgs/ImageMagick/template

```diff
@@ -1,28 +1,26 @@
 # Template file for 'ImageMagick'
 pkgname=ImageMagick
-version=7.0.11.12
-revision=3
+version=7.1.0.6
+revision=1
 _majorver=${version%.*}
 _patchver=${version##*.}
 wrksrc="${pkgname}-${_majorver}-${_patchver}"
 build_style=gnu-configure
-configure_args="--without-autotrace --with-wmf=yes
- --without-dps --without-fpx --without-gvc --without-jbig --with-gslib=yes
- --without-lqr --without-openexr --with-gs-font-dir=/usr/share/fonts/Type1
- --with-magick-plus-plus --with-modules --enable-shared --with-rsvg
- --with-dejavu-font-dir=/usr/share/fonts/TTF --enable-opencl --disable-static
- --with-openjp2"
+configure_args="--disable-static --enable-opencl --with-modules --with-gslib
+ --with-rsvg --with-wmf --with-dejavu-font-dir=/usr/share/fonts/TTF
+ --with-gs-font-dir=/usr/share/fonts/Type1"
 hostmakedepends="automake libtool pkg-config autoconf"
-makedepends="djvulibre-devel fftw-devel ghostscript-devel glib-devel lcms2-devel
```

Comment:
Rewrapping lines makes change unreadable, same for libmagick-devel depends.

Reply:
> .7 is out

I have updated to .8, restored the original line wrappings to make comparison easier, and restored the original distfile source (I originally just used the download location used on upstream's website, which is not necessary though).